src/libs/actions/Report.js

```diff
@@ -1109,6 +1119,84 @@ function updateLastReadActionID(reportID, sequenceNumber, manuallyMarked = false
     });
 }
 
+/**
+ * Gets all the report actions and updates the last read message
+ *
+ * @param {Number} reportID
+ */
+function openReport(reportID) {
+    const sequenceNumber = reportMaxSequenceNumbers[reportID];
```

Comment:
Yeah I thin we should remove this when we implement `OpenReport` b/c we don't need to pass a sequence number to it, it will automatically get it after loading the reportActions and set it to the latest

Reply:
This is fine for now if we are merging your PR before mine I can make some changes.